Corale.Colore/Core/Device.cs

```diff
@@ -40,7 +40,7 @@ public abstract class Device : IDevice
         /// <summary>
         /// Clears the current effect on the device.
         /// </summary>
-        public void Clear()
+        public virtual void Clear()
         {
             SetAll(Color.Black);
```

Comment:
I think I would prefer it to be marked abstract.  That way it can be implemented as per each device (if there needs to be discrepancies between devices).  It also means we won't forget to implement in new classes implementing `Device`.